packages/foreman/foreman/foreman.spec

```diff
@@ -819,8 +799,7 @@ Meta package to install asset pipeline support.
 %package plugin
 Summary: Foreman plugin support
 Group: Development/Libraries
-Requires: %{name} = %{version}-%{release}
```

Comment:
The plugin package still needs `foreman` installed so they can compile things like assets.